towhee/tests/mock_operators/add_operator_v2/add_operator_v2.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2021 Zilliz. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from typing import NamedTuple
+from towhee.operator import Operator, SharedType
+
+
+class AddOperatorV2(Operator):
+    """
+    Add operator that can take in undefined amount of inputs and saves results to '<input_name>_sum'
+    column of result dataframe.
+    """
+
+    def __init__(self, factor: int) -> None:
+        super().__init__()
+        self._factor = factor
+
+    def __call__(self, **kwargs):
```

Comment:
If the inputs and the outputs are dynamic, how should I create a pipeline without knowing the operator codes?

Reply:
We will use python interface which likes spark to create pipelines, so we must know the inputs and outputs col keys without the operator's codes.

Reply:
Filter and concat may directly use previous operator's col-keys, but other operators, I think must have their own col-keys.

Reply:
this was more for a setup of doing multiple adds in one operator. Add operator v1 only has num, only allowing addition of one column/df.